src/SpectatorView.Unity/MSBuild/settings.json

```diff
@@ -1 +1 @@
-{"autoGenerateEnabled":true}
\ No newline at end of file
+{"version":3,"autoGenerateEnabled":true,"dependenciesProjectGuid":"5a45860e-3555-4921-9cde-fb6d9945c4b8","assemblyCSharpGuid":"88a3e48c-339d-4416-92c3-307162279309","assemblyCSharpEditorGuid":"c9edf63e-d7b9-4f49-b5a0-4848a0205146","assemblyCSharpFirstPassGuid":"ccec8082-6754-4c7e-a742-8a94aa24b628","assemblyCSharpFirstPassEditorGuid":"2ac44e54-2ab1-47e5-b8e3-bd436d64e037","builtInPackagesFolderGuid":"2c55becf-70c1-4f9e-a90c-9e7f7688c9c4","importedPackagesFolderGuid":"91ab2fc0-d1d2-4bf1-8ce8-d160f619250b","externalPackagesFolderGuid":"3c35ba54-ff20-4034-a6e5-7696b24c5bf0","solutionGuid":"043e3c94-ee7e-405d-af1f-0fd50855f440"}
```

Comment:
Is it possible to prettify the json, or is this auto generated?

In general when checking in json/config, it's nicer to have them human readable (i.e. not just one line) so that things are also diffable going forward.

In the MRTK we had a massive json file that was just one line and it made diffing impossible. Then we just added new lines by running it through a formatter and it was a lot easier to deal with.

Reply:
This doesn't seem like something for this review, it may be worth filing an issue against this in the MSBuildForUnity repo.

@andreiborodin what are your thougts?

Reply:
Great idea! Please file an issue, or submit a PR ;)